packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/GeneratedCameraXLibrary.java

```diff
@@ -843,6 +919,185 @@ public void onDeviceOrientationChanged(@NonNull String orientationArg, Reply<Voi
             callback.reply(null);
           });
     }
+
+    public void onCameraError(@NonNull String errorDescriptionArg, Reply<Void> callback) {
+      BasicMessageChannel<Object> channel =
+          new BasicMessageChannel<>(
+              binaryMessenger,
+              "dev.flutter.pigeon.SystemServicesFlutterApi.onCameraError",
+              getCodec());
+      channel.send(
+          new ArrayList<Object>(Arrays.asList(errorDescriptionArg)),
+          channelReply -> {
+            callback.reply(null);
+          });
+    }
+  }
+
+  private static class PreviewHostApiCodec extends StandardMessageCodec {
+    public static final PreviewHostApiCodec INSTANCE = new PreviewHostApiCodec();
+
+    private PreviewHostApiCodec() {}
+
+    @Override
+    protected Object readValueOfType(byte type, ByteBuffer buffer) {
+      switch (type) {
+        case (byte) 128:
```

Comment:
Avoid magic numbers. Instead define them with a constant with a meaningful name. If this is generated code consider putting it in its own file with either documentation describing how to regenerate or at least that it is generated and where the source information live.